eogrow/core/eopatch.py

```diff
@@ -60,7 +60,7 @@ def _prepare_names(self) -> Tuple[bidict, Dict[str, BBox]]:
         """
         bbox_grid = self._area_manager.get_grid(add_bbox_column=True)
 
-        bbox_df: DataFrame = pandas.concat(bbox_grid, ignore_index=True)
+        bbox_df: DataFrame = pandas.concat([gdf.drop(columns="geometry") for gdf in bbox_grid], ignore_index=True)
```

Comment:
It was a pandas `DataFrame` even before, so I assume we've never really needed `geopandas` functionality on the bbox functionality, so I agree with the approach. 

Another - if "geometries" would be needed - would be to convert them to `sentinelhub.Geometry` objects.